### PR TITLE
Add the print-labels command to print a model's labels

### PR DESCRIPTION
**Pull request by danieldk**

### CODE REVIEW

**Comment by twuebi** [comment]
Why take the config to retrieve the labels file instead of taking the labels file as an argument?

**Comment by danieldk** [comment]
> Why take the config to retrieve the labels file instead of taking the labels file as an argument?

We need to know the encoder type to deserialize the numberer with the correct type parameter.

File: sticker-utils/src/subcommands/print_labels.rs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        E: SentenceEncoder,
+        E::Encoding: Clone + Eq + Hash + ToString,
+        Numberer<E::Encoding>: CborRead,
+    {
+        let labels: Numberer<E::Encoding> = config.labeler.load_labels().or_exit(
+            format!("Cannot load label file '{}'", config.labeler.labels),
+            1,
+        );
+
+        for idx in 1..labels.len() {
```
Comment (twuebi) [inline sticker-utils/src/subcommands/print_labels.rs:35]: Doesn't this produce a mismatch between the number of labels in the logits vs the number of labels here?

Comment (danieldk) [reply sticker-utils/src/subcommands/print_labels.rs:35]: Could you explain this a bit further?

sticker does not currently only use 0 for padding, but it also creates the classification layer using `labels.len()`, so there first logit is used for padding. Since we never backprop padding, its probability should normally be near-0. Though, it can never be predicted, because `top_k_predictions` excludes this logit/probability.

Comment (twuebi) [reply sticker-utils/src/subcommands/print_labels.rs:35]: If I load a graph and use this label list to map into the logits, I'd be one-off due to the missing padding, right?

Comment (danieldk) [reply sticker-utils/src/subcommands/print_labels.rs:35]: That's why it prints the index as well (starting at 0). I would also be fine with printing an artificial PADDING label before looping. Would that be good enough?

Comment (twuebi) [reply sticker-utils/src/subcommands/print_labels.rs:35]: Sure, I prefer being explicit here.

**Review by twuebi — APPROVED** [review]
